lib/Runtime/Library/JavascriptBigInt.h

```diff
@@ -0,0 +1,69 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#pragma once
+
+namespace Js
+{
+    class JavascriptBigInt sealed : public RecyclableObject
+    {
+
+    private:
+        Field(BOOL) m_isNegative;
+        Field(uint32*) m_digits;         // digits
+        Field(uint32) m_length;          // length
+
```

Comment:
Structure of a BigInt include sign (negative or not), array of uint32 digits, length (number of digits in the array)